portfolio_ahardman_stylesheet.css

```diff
@@ -0,0 +1,339 @@
+}
+main h3{
+	text-align: left;
+	font-size: 2rem;
+	margin: 2.5vh 0px 0px 0px;
+	padding: 0px;
+	border: 0px;
+}
+/* portfolio section */
+.portfolio{
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	background-color: white;
+}
+.portfolio .container{
+	width: 70%;
+	margin-top: 150px;
+	margin-bottom: 150px;
+}
+.portfolio h2{
+	display: flex;
+	align-items: center;
+	font-size: 2.5rem;
+	margin: 0px 0vh 60px 0vh;
+	padding: 0px;
+}
+.portfolio .container .builds_container{
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-items: center;
+}
+.build{
+	margin: 10px;
+	padding-bottom: 10px;
+	display: flex;
+	justify-content: space-around;
+	align-items: flex-end;
+	font-size: 1rem;
+	width: 200px;
+	height: 200px;
+	transition-duration: 0.6s;
+	border: 3px solid rgb(0, 164, 153);
+	border-radius: 5px;
+}
+.popup{
+	padding: 10px;
+	background-color: rgb(0, 204, 191);
+	border: 1px solid black;
+	border-radius: 5px;
+	font-size: 1rem;
+}
+.build.box1{
+	background-image: url('https://raw.githubusercontent.com/Mayden-Academy/FST-curriculum/master/MAYDEN%20ACADEMY_WEBSAFE_LOGO_06.15.png?token=AORF4IAOEYU56RY3T3FACIK6M6EHG');
+	background-size: 100%;
+	background-repeat: no-repeat;
+	background-position: 0% 20%;
+}	
+.build.box2{
+	background-image: url('https://github.com/amandahrdmn/pilot-webpage-exercise/blob/master/pilot_webpage_image.png?raw=true');
+	background-size: 100%;
+	background-repeat: no-repeat;
+	background-position: 0% 20%;
+}	
+.build.box_empty{
+	font-size: 1rem;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background-color: rgb(0, 204, 191);
+}
+/* ----------------------about me section---------------------- */
+.about_me{
+	padding-top: 150px;
+	padding-bottom: 150px;
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	align-items:center;
+	background-image: linear-gradient(to right,rgb(0, 164, 153),rgb(0, 204, 191));
+}
+.about_me .container{
+	width: 70%;
+}
+.about_me h2{
+	display: flex;
+	align-items: center;
+	font-size: 3rem;
+	padding: 0px;
+	margin: 0px;
+}
+.about_me .p_container{
+	display: flex;
+	flex-wrap: wrap;
+	margin-top: 20px;
+	width: 100%;
+}
+.p_container p{
+	width: 100%;
+	margin-top: 40px;
+	padding: 0px;
+	font-size: 1.5rem;
+}
+/* ----------------------footer---------------------- */
+footer{
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background-image: linear-gradient(to right,rgb(150,150,150),rgb(180,180,180));
+}
+footer .center.container{
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	padding: 1vw;
+}
+footer h3{
+	display: flex;
+	align-items: center;
+	text-align: center;
+	font-size: 1.5rem;
+	margin: 0px;
+	padding: 0px;
+}
+footer .center.container .right.container{
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+	margin-left: 2vw;
+}
+form{
+	display: flex;
+	align-items: center;
+	width: 100%;
+}
+.email_input{
+	margin-right: 1vw;
```

Comment:
generally using vw for margins and padding is a bad idea. suggest changing to px